### DISCUSSION TITLE
Qfieldcloud error not available primary keys

### DISCUSSION THREAD

**Discussion by jiptwo** [opengisch/QField#4455] [Q&A]
hi,
Sorry for my english .. i speak french
I have a problem with QfieldCloud community presently. 
My local Qfield project is working perfectly well and I was checking to see if everything was working in the community cloud before transferring my project to QfieldCloud.
With my ''poteaux'' layer When I take a photo in the ''poteaux info'' tab this works very well.
But when I want to put several photos in the ''Données couche enfant'' tab I have an error message ''Impossible d'ajouter l'entité enfant: les clés primaires du parent ne sont pas disponibles''
The column layer is linked to the ''childpoteau'' layer.
I leave you my project Qgis (8223xx Projet test.zip) for you to find the answer... 
otherwise I will continue to work locally unfortunately ;(

Tablet Dell Latitude 7230
Windows 11
Qfield 2.7.0
Qgis 3.32 on windows 10 pro
![Capture2](https://github.com/opengisch/QField/assets/35789788/0af0dac7-1911-41cb-b51f-92a11626df61)
[8223xx Projet test.zip](https://github.com/opengisch/QField/files/12051348/8223xx.Projet.test.zip)

**Comment by m-kuhn** [comment]
Hi, you can follow the [how to for adding multiple pictures](https://docs.qfield.org/how-to/pictures/#add-a-series-of-pictures-to-a-feature) especially the part with uuid applies here as well.

**Comment by jiptwo** [comment]
Good morning
this is exactly what I used!
Why does it work in local file and not in cloud??!!??
The problem is not solved!

**Comment by m-kuhn** [comment]
Can you post the configuration of the layers and relations?

**Reply by jiptwo** [reply]
My project is in my first message ... on Qgis 3.32 in the file .zip
